### PR TITLE
C13 soln

### PR DESCRIPTION
**Pull request by tildeee**
There are two commits: the implementation, and a commit where I outlined what and where to check against our rubric as comments throughout the file.

I'm unsure if the second (rubric) commit is useful, so I'd love feedback on that.

### CODE REVIEW

**Review by beccaelenzil — COMMENTED** [review]
This looks great. Great user experience. I would consider adding some new lines `\n` in the CLI to enhance user experience.

File: main.rb [hunk 1]
```diff
@@ -46,10 +46,13 @@ def get_new_user_planet
   fun_fact = gets.chomp
 
   new_planet = Planet.new(name, color, mass, distance, fun_fact)
-  
+
   return new_planet
 end
 
+
+# CODE REVIEW 7/7: Complex functionality is broken out into separate methods
```
Comment (beccaelenzil) [inline main.rb]: Should this be `CODE REVIEW 7/8` ?

**Review by beccaelenzil — APPROVED** [review]
I think the CODE REVIEW comments are helpful. I wonder if there's a way to prompt students to fully consider how the helper methods in main are really good choices for wrapping up functionality.

File: main.rb (hunk 1 above)
Comment (tildeee) [reply main.rb]: GOOD CATCH! Thank you!

**Pull request closed without merging**